### PR TITLE
Added the provision to run the live migration tests with the new YB CDC Logical replication connector

### PR DESCRIPTION
**Pull request by shubham-yb**
- Moved the Change ALTER to CREATE script to the top level
- Defined a new variable MOVE_PK_FROM_ALTER_TO_CREATE to invoke the script
- Added the provision to run the live migration tests with the new YB CDC Logical replication connector

### CODE REVIEW

**Comment by shubham-yb** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline-test/920/
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline-test/921/

File: migtests/scripts/live-migration-run-test.sh [hunk 1]
```diff
@@ -112,9 +112,8 @@ main() {
 		run_ysql yugabyte "CREATE DATABASE ${TARGET_DB_NAME}"
 	fi
 
-	if [ -x "${TEST_DIR}/add-pk-from-alter-to-create" ]
-	then
-		"${TEST_DIR}/add-pk-from-alter-to-create"
+	if [ "${SOURCE_DB_TYPE}" = "postgresql" ] ; then
```
Comment (priyanshi-yb) [inline migtests/scripts/live-migration-run-test.sh:115]: This will use this script for all the cases right?

Comment (priyanshi-yb) [reply migtests/scripts/live-migration-run-test.sh:115]: I think you should use `MOVE_PK_FROM_ALTER_TO_CREATE` here as well. to use these only when required

Comment (shubham-yb) [reply migtests/scripts/live-migration-run-test.sh:115]: Right. My miss. Thanks!

File: migtests/scripts/live-migration-run-test.sh [hunk 2]
```diff
@@ -181,7 +180,7 @@ main() {
 	trap - SIGINT SIGTERM EXIT SIGSEGV SIGHUP
 
 	step "Initiating cutover"
-	yb-voyager initiate cutover to target --export-dir ${EXPORT_DIR} --prepare-for-fall-back false --yes
+	yb-voyager initiate cutover to target --export-dir ${EXPORT_DIR} --prepare-for-fall-back false --yes --use-yb-grpc-connector f
```
Comment (priyanshi-yb) [inline migtests/scripts/live-migration-run-test.sh:183]: Why not using the `cutover_to_target` function here?

File: migtests/scripts/run-test.sh [hunk 3]
```diff
@@ -127,9 +127,8 @@ main() {
 		run_ysql yugabyte "CREATE DATABASE ${TARGET_DB_NAME}"
 	fi
 
-	if [ -x "${TEST_DIR}/add-pk-from-alter-to-create" ]
-	then
-		"${TEST_DIR}/add-pk-from-alter-to-create"
+	if [ "${SOURCE_DB_TYPE}" = "postgresql" ] ; then
```
Comment (priyanshi-yb) [inline migtests/scripts/run-test.sh:130]: "${MOVE_PK_FROM_ALTER_TO_CREATE}" = true?

**Review by priyanshi-yb — APPROVED** [review]
LGTM

**Pull request merged**